src/components/ambient-light.js

```diff
@@ -0,0 +1,27 @@
+AFRAME.registerComponent("ambient-light", {
+  schema: {
+    color: { type: "color" },
+    intensity: { default: 1.0 }
+  },
+
+  init() {
+    const el = this.el;
+    this.light = new THREE.AmbientLight();
+    this.el.setObject3D("ambient-light", this.light);
+    this.el.sceneEl.systems.light.registerLight(el);
```

Comment:
Since we dont own this system it feels a bit weird to be registering with it. From what I understand you are doing this to remove the default lights so maybe we should just set `defaultLightsEnabled` to `false` on the light system.  Not a huge deal either way.